onadata/libs/tests/serializers/test_user_profile_serializer.py

```diff
@@ -75,3 +78,28 @@ def test_get_temp_token_recreates_if_expired(self):
         is_expired = expired(temp_token.created)
 
         self.assertFalse(is_expired)
+
+
+class TestUserProfileSerializer(TestAbstractViewSet):
+
+    def test_metadata_view_for_owner_only(self):
```

Comment:
How is this test related to the issue being addressed by this pull-request on password edits?

Reply:
The field inclusion had been taken care of in different tests. This test is to ensure that the last_password_edit field can only be viewed by the account owner, for security reasons.